test/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/GetRecordsConnectorSpec.scala

```diff
@@ -64,7 +64,7 @@ class GetRecordsConnectorSpec extends BaseConnectorSpec with GetRecordsDataSuppo
       when(requestBuilder.execute[Either[Result, GetEisRecordsResponse]](any, any))
         .thenReturn(Future.successful(Left(BadRequest("error"))))
 
-      val result = await(connector.fetchRecord(eori, recordId, correlationId, true))
+      val result = await(connector.fetchRecord(eori, recordId, correlationId, appConfig.hawkConfig.getRecordsUrl))
```

Comment:
I would rename the test and remove the reference to Hawk or Pega in here as the connector does not know  anything about Hawk or Pega configuration. For this reason may better to pass any string as url parameter in the function